Include/pytime.h

```diff
@@ -29,7 +29,10 @@ typedef enum {
     _PyTime_ROUND_CEILING=1,
     /* Round to nearest with ties going to nearest even integer.
        For example, used to round from a Python float. */
-    _PyTime_ROUND_HALF_EVEN
+    _PyTime_ROUND_HALF_EVEN=2,
+    /* Round away from zero 
+       For example, used for timeout if negative values are allowed. */
+    _PyTime_ROUND_UP=3
```

Comment:
Since rounding modes changed many times last 5 years and are very tricky to choose, I suggest to add an alias:

_PyTime_ROUND_TIMEOUT=_PyTime_ROUND_UP

_PyTime_ROUND_TIMEOUT should be used for timeouts.

> "For example, used for timeout if negative values are allowed."

I suggest to elaborate the behaviour for negative values.

"For example, used for timeout. _PyTime_ROUND_CEILING rounds -1e-9 to 0 milliseconds which causes bpo-31786 issue. _PyTime_ROUND_UP rounds -1e-9 to -1 millisecond which keeps the timeout sign as expected. select.poll(timeout) must block for negative values."

Reply:
Corrected in a21d10d.